src/screens/SectionScreen.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { SafeAreaView, StyleSheet, Text, View } from 'react-native';
+import COLORS from '../assets/colors';
+
+const SectionScreen = () => {
+  return (
+    <SafeAreaView style={styles.background}>
+      <View style={styles.container}>
+        {/*TODO: if statement checking database data for existing section title*/}
+        <Text style={styles.bottomText}>Onderdeel</Text>
+      </View>
+      <View style={styles.containerDescription}>
+        <Text style={styles.descriptionTitle}>Beschrijving</Text>
+        <View style={styles.descriptionLine} />
+        {/*TODO: if statement checking database data for existing section description*/}
+        <Text style={styles.descriptionText}>Er is voor dit onderdeel geen beschrijving </Text>
+        <View style={styles.descriptionLineQuestions} />
+      </View>
+    </SafeAreaView>
+  );
+};
+
+const styles = StyleSheet.create({
+  background: {
+    backgroundColor: COLORS.white,
+    height: '100%',
+    width: '100%',
+  },
+  container: {
+    backgroundColor: COLORS.darkBlue,
+    justifyContent: 'center',
+    height: '20%',
+  },
+  bottomText: {
+    color: COLORS.white,
+    fontSize: 40,
+    fontFamily: 'Muli-SemiBold',
+    textAlign: 'center',
+    alignContent: 'center',
+  },
+  descriptionTitle: {
+    top: '10%',
+    left: '10%',
+    color: 'black',
```

Comment:
Hier even `color: COLORS.black` van maken, dan houden we alle kleuren lekker consistent bij elkaar